{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "ffd66791_ded20140",
        "filename": "nvme_manager.cpp",
        "patchSetId": 1
      },
      "lineNbr": 560,
      "author": {
        "id": 1000388
      },
      "writtenOn": "2023-02-14T17:38:38Z",
      "side": 1,
      "message": "nit: let\u0027s declare this closer to where it gets used (at line 591)",
      "range": {
        "startLine": 560,
        "startChar": 0,
        "endLine": 560,
        "endChar": 41
      },
      "revId": "170042ebfd59459917a952bc114f509b0d5ddce6",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "1437fb8a_84cf7911",
        "filename": "nvme_manager.cpp",
        "patchSetId": 1
      },
      "lineNbr": 574,
      "author": {
        "id": 1000388
      },
      "writtenOn": "2023-02-14T17:38:38Z",
      "side": 1,
      "message": "Hmm... does this make sense?\n\nShould we behave closer to when it was not successful (the else statement) and return?\n\nI guess we already set success to false above - do we even want to go through the below code? Should we return at line 571 after setting success\u003dfalse?",
      "range": {
        "startLine": 574,
        "startChar": 16,
        "endLine": 574,
        "endChar": 29
      },
      "revId": "170042ebfd59459917a952bc114f509b0d5ddce6",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}